google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -44,7 +44,8 @@ appengine.flex.config.generation.io.error=There was an error generating the {0}.
 appengine.config.project.panel.title=Project: {0}
 appengine.flex.config.user.specified.artifact.title=Select a JAR or WAR file
 appengine.flex.config.user.specified.artifact.error=Browse to a JAR or WAR file.
-appengine.flex.config.custom.deploy.title=App Engine Flexible Environment Configuration.
+appengine.flex.config.custom.deploy.title=App Engine Flexible Environment Configuration
+appengine.flex.config.custom.configfiles.error=Provide app.yaml and Dockerfile.
```

Comment:
To keep it consistent with the jar/war error message, how about:

"Browse to an app.yaml file."
"Browse to a Dockerfile."